bin/chelsea

```diff
@@ -34,6 +34,7 @@ opts =
       o.bool '-v', '--verbose', 'Make chelsea only output vulnerable third party dependencies for text output (default: true)', default: false 
       o.string '-t', '--format', 'Choose what type of format you want your report in (default: text) (options: text, json, xml)', default: 'text'
       o.bool '-b', '--iq', 'Use Nexus IQ Server to audit your project'
+      o.string '-s', '--stage', 'Specify Nexus IQ Stage'
```

Comment:
Might be good to set a `default: 'develop'`?

Reply:
Good idea. I went with `build` as the default, since that is IQ's default, and is the same behavior as prior Chelsea releases.  b06239b